build/packaging/linux/conf/kscnd.conf

```diff
@@ -47,6 +47,9 @@ SC_ANCHORING=0
 SC_ANCHORING_PERIOD=1
 SC_TX_LIMIT=1000
 
+SC_VTRECOVERY=1
+SC_VTRECOVERY_INTERVAL=5
+
```

Comment:
`ks*nd.conf` file still has `VTRECOVERY=1` as a default configuration. If you want to change `VTRECOVERY` to `SC_VTRECOVERY` in `ks*nd` file, please remove `VTRECOVERY` in `ks*nd.conf` or allow both `VTRECOVERY` and `SC_VTRECOVERY` in `ks*nd` file.

Reply:
Thank you. Removed and added brief comments.